src/item/TieredTool.php

```diff
@@ -61,4 +63,8 @@ public function getFuelTime() : int{
 	public function isFireProof() : bool{
 		return $this->tier === ToolTier::NETHERITE;
 	}
+
+	public function isValidRepairMaterial(Item $material) : bool{
```

Comment:
i don't like this name, feel that the material terminology doesn't fit, instead i would name it something like canBeRepairedBy or isRepairedBy

Reply:
Looks more like french tbh 😂

Reply:
This is the name used in the wiki to refer to materials.
But just specifying, `canBeRepairedBy` can lead to confusion in thinking that it can be both the materials and the tool/armour itself. Whereas what we're expecting here is just the materials